.storybook/preview.ts

```diff
@@ -1,5 +1,5 @@
-import type { Preview } from "@storybook/react";
 import React from "react";
```

Comment:
@2div Do we really need this?

Reply:
In preview.ts file when remove:  

import React from "react";
import type { Preview } from "@storybook/react";

i will get errors so that i keep them, please see the below screenshot , commented the 2 imports statements:

![image](https://github.com/dest-nft/dest.sh/assets/96177680/7bf4ae3b-266c-4bb7-8a4e-bad81597acc4)

Reply:
@2div In `preview.ts` `React.createElement` is used

```ts
  decorators: [
    (story) => {
      return React.createElement(
        "div",
        {
          className: `${clashDisplay.variable} ${outfit.variable} font-outfit`,
        },
        story(),
      );
    },
  ],
```

You should import
```ts
import type { Preview } from "@storybook/react";
import { createElement } from "react";
```
and use it instead of `React.createElement`

Reply:
@yura2100 i have applied suggested changes , please reviews it.  Thank you